publications-2021.bib

```diff
@@ -78,13 +78,16 @@ @InProceedings{MaWi21
 url       = {https://www.scipedia.com/public/Mang_Wick_2021a}
 }
 
-@misc{mang2021mixed,
-title={A mixed phase-field fracture model for crack propagation in punctured EPDM strips},
-author={Katrin Mang and Andreas Fehse and Nils Hendrik Kr{\"o}ger and Thomas Wick},
-year={2021},
-eprint={2104.14826},
-archivePrefix={arXiv},
-primaryClass={math.NA}
+@article{Mang2021,
+  doi = {10.1016/j.tafmec.2021.103076},
+  url = {https://doi.org/10.1016/j.tafmec.2021.103076},
+  year = {2021},
+  month = aug,
+  publisher = {Elsevier {BV}},
+  pages = {103076},
```

Comment:
are these page numbers right? This looks like its just the last part of the DOI.